package.json

```diff
@@ -30,37 +30,37 @@
     "@astrojs/partytown": "^1.0.1",
     "@astrojs/prism": "^1.0.2",
     "@chainlink/design-system": "^0.1.18",
-    "@metamask/detect-provider": "^1.2.0",
-    "@metamask/providers": "^9.1.0",
+    "@metamask/detect-provider": "^2.0.0",
+    "@metamask/providers": "^10.2.1",
     "@nanostores/preact": "^0.3.1",
     "clipboard": "^2.0.11",
     "ethers": "^5.7.2",
-    "focus-trap-react": "^10.0.0",
-    "github-slugger": "^1.4.0",
+    "focus-trap-react": "^10.0.1",
+    "github-slugger": "^2.0.0",
     "nanostores": "^0.7.1",
     "preact": "^10.11.3",
     "query-string": "^7.1.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
```suggestion
```

can be remove once this gets merged https://github.com/smartcontractkit/documentation/pull/1074

Reply:
Removed